package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "compliance-viewer",
+  "version": "0.0.1",
+  "description": "A viewer for compliance information.",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/18F/compliance-viewer.git"
+  },
+  "keywords": [
+    "18f",
+    "compliance",
+    "s3"
+  ],
+  "author": "",
+  "license": "SEE LICENSE IN LICENSE.md",
+  "bugs": {
+    "url": "https://github.com/18F/compliance-viewer/issues"
+  },
+  "homepage": "https://github.com/18F/compliance-viewer#readme",
```

Comment:
FYI you can leave most of these fields out if you use `"private": true`.

https://docs.npmjs.com/files/package.json#private